terraform/aws/vpc/main.tf

```diff
@@ -0,0 +1,22 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "2.15.0"
+
+  name               = "${var.cluster_name}-vpc"
+  cidr               = "10.18.0.0/16"
+  azs                = ["${var.region}a"]
+  enable_nat_gateway = true
+  enable_vpn_gateway = true
+
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  public_subnets = ["10.18.128.0/20"]
```

Comment:
Hardcode. And can be computed from `cidr`

Here example:

```terraform
variable "base_network" {
  default = "10.21.192.0/19"
}

locals {
  vpc_cidr = cidrsubnet(var.base_network, 6, 6) # 10.21.195.0/25
}

module "vpc" {
  public_subnets  = [cidrsubnet(local.vpc_cidr, 2, 0), cidrsubnet(local.vpc_cidr, 2, 2)]
  private_subnets = [cidrsubnet(local.vpc_cidr, 2, 1), cidrsubnet(local.vpc_cidr, 2, 3)]
}
```
https://www.terraform.io/docs/configuration/functions/cidrsubnet.html
https://www.terraform.io/docs/configuration/functions/cidrsubnets.html

Reply:
done